test/language/expressions/optional-chaining/optional-call-preserves-this.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2019 Sony Interactive Entertainment Inc.
```

Comment:
```suggestion
// Copyright (C) 2019 Sony Interactive Entertainment Inc. All rights reserved.
```

this would be a technical fix for the linter